docs/docs/clients/rest.md

```diff
@@ -261,6 +261,120 @@ create_feature_state_response = session.post(create_segment_override_url, json=f
 assert create_feature_state_response.status_code == 201
 ```
 
+### Create identity overrides
+
+Creating identity overrides varies depending on if you are using Flagsmith SaaS or a different Flagsmith environment.
+
+<details>
+
+<summary>Flagsmith SaaS</summary>
+
+Creating an identity override requires the following information:
+
+* Environment ID to create the override in
+* Identifier to create the override for
+* Name of the feature to create the override for
+* Desired feature state
+
+To create an identity override, use the
+[Update Edge Identity Feature State endpoint](https://api.flagsmith.com/api/v1/docs/#/operations-api-api_v1_environments_edge-identities_list).
+For example, the following request would enable the feature named `custom_background_colour` with a value of `blue`
+for the identity `my_user_id`:
+
+```
+curl --request PUT \
+  --url https://api.flagsmith.com/api/v1/environments/environments/YOUR_ENVIRONMENT_ID/edge-identities-featurestates \
+  --header 'Accept: application/json' \
+  --header 'Authorization: Token YOUR_ADMIN_API_KEY' \
+  --header 'content-type: application/json' \
+  --data '{"enabled":true,"feature":"custom_background_colour","feature_state_value":"blue", "identifier":"my_user_id"}'
+```
+
+This will create the override if it doesn't exist, or update it if it does (upsert).
+
+</details>
+
+<details>
+
+<summary>Self-hosted and private cloud</summary>
+
+Creating an identity override in non-SaaS environments requires additional information compared to Flagsmith SaaS:
+
+* Internal ID of the feature to override
+* Internal ID of the target identity
+* ID of the identity feature state to update, if an override already exists
+
+Make sure you have [JSON View](#json) enabled.
+
+To obtain the feature's internal ID, open the feature in the Flagsmith dashboard, and expand the "JSON Data: Feature"
+section. The feature's internal ID is the `id` field of this JSON object.
+
+To obtain the internal IDs of the target identity, browse to the target identity from the Identities section in the
+Flagsmith dashboard. The identity's internal ID is displayed in the URL, which is `1234` in this example:
+
+```
+https://flagsmith.example.com/project/5/environment/AbCxYz/users/my_user_id/1234
```

Comment:
```suggestion
https://flagsmith.example.com/project/5/environment/AbCxYz/users/my_identifier/1234
```

nit: I think let's be more generic here. We use identifier elsewhere in the documentation, so let's stick with it here too.